docs/import_export/README.md

```diff
@@ -0,0 +1,109 @@
+# Import/Export Documentation for uPortal 5
+
+## Overview
+
+The Import/Export tool in uPortal 5 allows users to transfer configuration data and settings between instances, facilitating data migration and backup.
+
+## Data tasks
+
+- **dataImport**: Adds to or updates the portal database for the entities defined in the specified XML file(s). Requires -Dfile={path-to-file} or -Ddir={path-to-directory}
+
+- **dataList**: With no arguments (-Dtype={entity-type}) lists all supported portal data types and the operations (export, delete) supported for each.
+
+- **dataExport**: Creates XML files representing the requested entities and writes them to the specified file system location. Parameters: -Ddir={path-to-directory} -Dtype={entity-type} [-Dsysid={entity-identifier}]
+
+- **dataDelete**: Deletes the specified entity. Requires -Dtype={entity-type} and -Dsysid={id}
+
+- **dataInit**: Drop and recreate uPortal tables and reimport data
+
+
+## Running Tasks via Gradle
+To run the Import/Export tool via Gradle, navigate to the uPortal project directory and execute
```

Comment:
```suggestion
To run the Import/Export tool via Gradle, execute the following from uPortal-start
```

Interestingly, the tasks are not available in this repo, but in the uPortal-start repo